packages/test-base/src/androidTest/kotlin/io/realm/kotlin/test/shared/MutableRealmTests.kt

```diff
@@ -675,4 +675,28 @@ class MutableRealmTests {
             }
         }
     }
+
+    @Test
+    fun deleteAll() {
+        realm.writeBlocking {
+            for (i in 0..9) {
+                copyToRealm(Parent())
```

Comment:
I would probably cut down on the number of types in here. Having 5 seems overkill, especially if there are no differences between them

I would try to aim to test different schema types, i.e. `RealmObject`, `RealmObject` with primary key and an `EmbeddedRealmObject`